regression-test/suites/trino_p0/sql/group_by/runConstantGroupBy_order.sql

```diff
@@ -1 +1,4 @@
-select 2 from nation group by 1
+select 2 from nation group by 1;
+select "oneline" from nation group by "I am a const, but not to be removed";
```

Comment:
we need to add a regression test on empty table to set scalar agg and vector agg's results.

Reply:
done